There is an error only on `win `platforms:
```
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\ci\openssl_1639585494239\_test_env\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "C:\ci\openssl_1639585494239\_test_env\lib\urllib\request.py", line 525, in open
    response = self._open(req, data)
  File "C:\ci\openssl_1639585494239\_test_env\lib\urllib\request.py", line 543, in _open
    '_open', req)
  File "C:\ci\openssl_1639585494239\_test_env\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "C:\ci\openssl_1639585494239\_test_env\lib\urllib\request.py", line 1393, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "C:\ci\openssl_1639585494239\_test_env\lib\urllib\request.py", line 1352, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)>
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -48,7 +48,8 @@ requirements:
 
 test:
   requires:
-    - python 3.7
+    - python 3.7  # [not (osx and arm64)]
+    - python 3.8  # [osx and arm64]
     - six
   commands:
     - copy NUL checksum.txt        # [win]

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -56,8 +56,8 @@ test:
     - copy NUL checksum.txt        # [win]
     - touch checksum.txt           # [unix]
     - openssl sha256 checksum.txt
-    - python -c "from six.moves import urllib; urllib.request.urlopen('https://pypi.org')"
-
+    - python -c "from six.moves import urllib; urllib.request.urlopen('https://pypi.org')"  # [unix]
+    - python -c "import certifi; import ssl; import urllib.request as urlrq; urlrq.urlopen('https://pypi.org', context=ssl.create_default_context(cafile=certifi.where()))"  # [win]
 
 about:
   home: http://www.openssl.org/

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -48,6 +48,7 @@ requirements:
 
 test:
   requires:
+    - certifi  # [win]
     - python 3.7  # [not (osx and arm64)]
     - python 3.8  # [osx and arm64]
     - six

